### PR TITLE
40460 - [CDP] - Consolidate LTR and MCP Action References

### PR DESCRIPTION
**Pull request by OptionSelect**
## Description
Updated actions to point to their combined counterparts, refactored LTR slightly to use hooks.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#40460

## Testing done
Ran cypress tests and components still rendered effectively in each experience

## Screenshots
<details>
<summary>Click to Open</summary>

image
image
</details>

## Acceptance criteria
- [ X ] Components still render as expected while actions in experience point to combined portal.

### CODE REVIEW

File: src/applications/debt-letters/components/DebtLettersWrapper.jsx [hunk 1]
```diff
@@ -13,15 +12,17 @@ const DebtLettersWrapper = ({
...
-  getDebtLetters,
 }) => {
+  const dispatch = useDispatch();
+
   useEffect(
     () => {
       if (showDebtLetters) {
-        getDebtLetters();
+        const generateGetDebtLetters = () => fetchDebtLetters(dispatch);
+        dispatch(generateGetDebtLetters);
```
Comment (jerekshoe) [inline src/applications/debt-letters/components/DebtLettersWrapper.jsx:22]: Is there a reason you can't just do this:
```
dispatch(fetchDebtLetters);
```

Comment (OptionSelect) [reply src/applications/debt-letters/components/DebtLettersWrapper.jsx]: Turns out there was not! I was unaware of that syntax and have updated the code to reflect the suggestion! The tests still passed with that new dispatch in place.

File: src/applications/medical-copays/containers/MedicalCopaysApp.jsx [hunk 2]
```diff
@@ -27,7 +27,8 @@ const MedicalCopaysApp = ({ children }) => {
   useEffect(
     () => {
       if (userLoggedIn) {
-        dispatch(getStatements());
+        const generateGetStatements = () => getStatements(dispatch);
+        dispatch(generateGetStatements);
```
Comment (jerekshoe) [inline src/applications/medical-copays/containers/MedicalCopaysApp.jsx:31]: You can also change this to:
```
dispatch(getStatements);
```

**Review by jerekshoe — APPROVED** [review]

File: src/applications/debt-letters/components/DebtLettersWrapper.jsx (hunk 1 above)
Comment (jerekshoe) [reply src/applications/debt-letters/components/DebtLettersWrapper.jsx]: Great, glad that worked!

File: src/applications/medical-copays/containers/MedicalCopaysApp.jsx (hunk 2 above)
Comment (OptionSelect) [reply src/applications/medical-copays/containers/MedicalCopaysApp.jsx]: You are absolutely right, thank you for catching that as well. I've just updated the same and confirm that the tests continue to pass after refactor.

**Review by mdewey — APPROVED** [review]
LGTM

**Pull request merged**